### PR TITLE
feat: add version package

### PR DESCRIPTION
**Pull request by busser**
## Features

- Add a `version` package containing Clamp's version information and helper print functions.

### CODE REVIEW

File: version/version.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	"os"
+)
+
+var (
+	// Version is Clamp's version.
+	Version = "unknown version"
+	// Commit is the hash of the commit used to build Clamp.
+	Commit = "unknown commit"
+	// Date is when Clamp was built.
+	Date = "unknown build date"
```
Comment (JulienBreux) [inline version/version.go]:
```suggestion
	Version = "dev"
	// Commit is the hash of the commit used to build Clamp.
	Commit = "n/a"
	// Date is when Clamp was built.
	Date = "n/a"
```

**Review by JulienBreux — CHANGES_REQUESTED** [review]

File: version/version.go (hunk 1 above)
Comment (busser) [reply version/version.go]: Fixed. The default values are now back to what they were originally.

**Review by JulienBreux — DISMISSED** [review]
Soon merge

**Comment by busser** [comment]
@JulienBreux I noticed a copy/paste error I'd made in `.goreleaser.yml`, so I added a fixup to my first commit. Sorry for the untimely fix 😕

**Comment by busser** [comment]
This PR does not add any actual features. All it does is put version information into a dedicated package.

**Pull request closed without merging**